### DISCUSSION TITLE
ActionButtons not working in ActionGroup

### DISCUSSION THREAD

**Discussion by conormdowney** [adobe/react-spectrum#2125] [Q&A] [answered]
I have the following code:

```
<Provider theme={defaultTheme}>
                <ActionGroup 
                    overflowMode="collapse" 
                    maxWidth={10}>
                    <ActionButton>Icon + Label</ActionButton>
                    <ActionButton>Icon + Label2</ActionButton>
                </ActionGroup>
               
            </Provider>
```

all i want it to do is render as the collapsed button and open to show those options when i click it. But instead my application blows up.
If I change the ActionButtons to Items it works fine. The ActionGroup page says it is a grouping of Action Buttons but then none of hte examples use ActionButtons. Can anyone tell me what Im doing wrong?

**Comment by LFDanLu** [comment, marked as answer]
ActionGroup follows the [collections api](https://react-spectrum.adobe.com/react-stately/collections.html) and thus accepts `Item`s or a [function that returns `Item`](https://react-spectrum.adobe.com/react-spectrum/ActionGroup.html#content). The ActionGroup component itself then handles rendering the ActionButtons internally.
